code/game/objects/items/devices/traitordevices.dm

```diff
@@ -237,14 +237,12 @@ effective or pretty fucking useless.
 	name = "radio jammer"
 	desc = "Device used to disrupt nearby radio communication."
 	icon_state = "jammer"
-	var/active = FALSE
-	var/range = 12
+
+/obj/item/device/jammer/Initialize()
+	AddComponent(/datum/component/jammer, 12, src)
 
 /obj/item/device/jammer/attack_self(mob/user)
```

Comment:
Lol woops i forgot i need to remove this, ill do that tommorow because im sleeping now zz